src/prefect/_internal/compatibility/experimental.py

```diff
@@ -177,3 +177,14 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def enabled_experiments() -> Set[str]:
+    """
+    Return the set of all enabled experiments.
+    """
+    return {
+        name[len("PREFECT_EXPERIMENTAL_ENABLE_") :].lower()
```

Comment:
Let's open a maintenance ticket to move this string prefix into a constant so we don't repeat it.